src/app/view/timestamp-editor/timestamp-editor.component.html

```diff
@@ -3,77 +3,90 @@ <h4>For Transport-Call {{ transportCall.transportCallSequenceNumber}}
 <div class="headline">
   <p-card styleClass="card-secondary">
     <div class="p-fluid p-formgrid grid">
-      <div class="p-field col-12 md:col-4">
+      <div class="md:col-4">
         <label class="label-compressed" for="vesselName">{{'general.transportCall.table.vesselName.label'| translate}}
           : </label>
         <span id="vesselName"> {{transportCall.vessel.vesselName ?? "(Name not known)"}}</span>
       </div>
-      <div class="p-field col-12 md:col-4">
+      <div class="md:col-4">
         <label class="label-compressed" for="portOfCall">{{'general.port.ofCall'| translate}}: </label>
         <span id="portOfCall" *ngIf="transportCall.portOfCall != null">
           {{transportCall.portOfCall.UNLocationCode}} </span>
       </div>
-      <div class="p-field col-12 md:col-4">
+      <div class="md:col-4">
         <!-- Placeholder for additional information -->
       </div>
-      <div class="p-field col-12 md:col-4">
+      <div class="md:col-4">
         <label class="label-compressed" for="vesselIMO">{{'general.transportCall.table.vesselImo.label'| translate}}
           : </label>
         <span id="vesselIMO"> {{transportCall.vessel.vesselIMONumber}}</span>
       </div>
+      <div class="md:col-4" *ngIf="respondingToTimestampInfo != null">
+        <!-- Placeholder for additional information when rejecting timestamp -->
+        <div *ngIf="parseTimestampResponseStatus() == 'ACCEPTED'">
```

Comment:
Can we use the enum here rather than `'ACCEPTED'` as a string?  (Ditto for future/further cases)

Reply:
Not to my knowledge, but will have another look at it

Reply:
Alternative, we can rewrite it to `isAcceptResponseStatus()` and have the condition occur inside the .ts file (where we can use the enum).

Reply:
Apparently it is possible, so I made use of the enums in the template directly.